glRemixRenderer/shaders/rt/raytracing.hlsl

```diff
@@ -211,10 +213,16 @@ float3 transform_to_world(float3 local_dir, float3 N)
              (curr_light.linear_attenuation * light_dist) +
              (curr_light.quadratic_attenuation * light_dist * light_dist));
 
-    const float n_dot_l = max(dot(n_world, light_vec), 0.0);
-    float3 ambient = mat.ambient.rgb * albedo;
-    float3 diffuse = mat.diffuse.rgb * albedo * n_dot_l * light_color;
-    float3 color = albedo; // float3(1.0, 1.0, 1.0); // diffuse;
+        // diffuse
```

Comment:
@glRemix/glremix-team @anya0402 Can you double check that I resolved this merge conflict correctly?